src/components/inputs/datetimePicker/DateTimePicker.tsx

```diff
@@ -148,7 +152,7 @@ const DateTimePicker = React.forwardRef<DatePicker, DateTimePickerProps>(functio
 	);
 
 	useEffect(() => {
-		updateDateTimeState({ type: 'SAVE_AND_UPDATE', value: defaultValue });
+		updateDateTimeState({ type: 'DEFAULT_VALUE_CHANGED', value: defaultValue });
```

Comment:
If updateDateTimeState dependency is not a triggering dependency, it could be simplified like this and the updateDateTimeState dependency can be removed along with the new case added in the above part.
This could avoid further unwanted rerenders
```suggestion
		dateTimeRef.current = newValue;
		setDateTime(newValue);
```